src/gui/mrview/window.cpp

```diff
@@ -2012,7 +2013,7 @@ namespace MR
             return;
           }
 
-          assert ("shouldn't reach here!" && false);
+          assert (opt.opt->is ("info") or opt.opt->is ("debug") or ("shouldn't reach here!" && false));
```

Comment:
[seems to work](https://qph.ec.quoracdn.net/main-qimg-d2762148238117d097dba8367d42bf33-c)